packages/core/src/Stores/traders-hub-store.js

```diff
@@ -735,17 +742,23 @@ export default class TradersHubStore extends BaseStore {
     }
 
     async getTotalBalance(accounts, base_currency) {
```

Comment:
@shayan-deriv We can extract this into a reusable hook and just use it inside the component instead of injecting the `exchange_rate` value into this store, Maybe that would be out of this PR scope so we can leave it as it is and handle it in a follow-up PR later 🙇🏻 

---

Please remove the commented codes 🙏🏻